### DISCUSSION TITLE
Number of Posts

### DISCUSSION THREAD

**Discussion by tderflinger** [neet/masto.js#749] [Q&A] [answered]
Hi,
Thanks for the great work! Probably I just haven't looked thoroughly enough, but how do I get the total number of posts? This information is also displayed on the profile in the web application.

I could not find any suitable method in the documentation.

Thanks for any hint!
-Thomas

**Comment by neet** [comment, marked as answer]
According to Mastodon official documentation

> `statuses_count`
> **Description:** How many statuses are attached to this account.
> https://docs.joinmastodon.org/entities/Account/#statuses_count

Masto.js corresponding to this field is as follows

```ts
const masto = await login({...});
const account = await masto.v1.accounts.fetch("<ACCOUNT ID>");
account.statusesCount;
```

**Reply by tderflinger** [reply]
Thanks!
